docs/README.md

```diff
@@ -65,3 +66,9 @@
 | [Why SRT Was Created](misc/why-srt-was-created.md) |         [misc](misc/)         | [why-srt-was-created.md](misc/why-srt-was-created.md) | Background and history of SRT. See also<br />[Section 1. Introduction](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00#section-1) of the [SRT RFC](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00). |
 | <img width=200px height=1px/>                      | <img width=100px height=1px/> | <img width=200px height=1px/>                         | <img width=500px height=1px/>                                |
 
+
+[srt-rfc]: https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00
```

Comment:
Please revert most of these changes. 

- I guess that it would be useful to only leave a reference for `srt-rfc` and then in the text you can refer to a separate sections like `srt-rfc#section-1` (if it works, I hope so). 
- If we do such a change, it's better to go through the whole document and update the references.
- For other links, let us please use the links without shortcuts.
- Please change "SRT RFC" to "SRT RFC Draft" throughout the document. Or I can do it later within a separate PR.

Reply:
> I guess that it would be useful to only leave a reference for srt-rfc and then in the text you can refer to a separate sections like srt-rfc#section-1 (if it works, I hope so).

It was the first what I tried, but it doesn't work this way.